worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -671,8 +671,14 @@ private[celeborn] class Worker(
     val resourceConsumptionSnapshot = storageManager.userResourceConsumptionSnapshot()
     val userResourceConsumptions =
       workerInfo.updateThenGetUserResourceConsumption(resourceConsumptionSnapshot.asJava)
-    resourceConsumptionSnapshot.foreach { case (userIdentifier, _) =>
+    resourceConsumptionSnapshot.foreach { case (userIdentifier, userResourceConsumption) =>
```

Comment:
It would be better not persist resourceConsumption into ratis When worker heartbeat subResourceConsumption

Reply:
Make sense.